### ISSUE TITLE
[FIREFOX] The add-on is no longer available

### ISSUE THREAD

**Issue by walrus543** [rohanb10/snoozz-tab-snoozing#112]
## Description
Go to the official link: https://addons.mozilla.org/en-US/firefox/addon/snoozz/

The add-on is no longer available. Did you remove it? Will it be added again?
Thank you.

**Comment by RicardoCapricciosa** [comment]
Yes please add it again, it seems even my installation was automatically removed from firefox.

**Comment by guardiaopt** [comment]
Firefox is getting less and less friendly with developers, I've also had similar problem, the solution was to put my extension on my site.

**Comment by walrus543** [comment]
For the time being, would it be possible to have a xpi/zip file to install the addon manually?
https://extensionworkshop.com/documentation/publish/distribute-sideloading/

**Comment by concese** [comment]
Today I had to reinstall firefox and all my extensions. 
It's still not available on firefox addons repository. 

Any news on this?

**Comment by RicardoCapricciosa** [comment]
Anything? I'm desperately missing this addon for my firefox. Is there another way to directly maybe install it with the code from here? If yes how?

**Comment by gmunguia** [comment]
For anyone interested, I've forked the project and re-published:

Add-on: https://addons.mozilla.org/en-US/firefox/addon/snooze-tabs-windows-fork/
Source: https://github.com/gmunguia/snoozz-tab-snoozing

@rohanb10, I'd be happy to collaborate to merge the fork back into this repo.

**Comment by RicardoCapricciosa** [comment]
@gmunguia Thank you so much! This made my day.

**Comment by walrus543** [comment]
Thanks @gmunguia  So it wasn't a validation problem with Mozilla. Still curious to understand why it has be removed...

**Comment by matheusfaustino** [comment]
@Primokorn they did not explain to you why it was removed? o0

**Comment by walrus543** [comment]
@matheusfaustino no I have no explanation.

**Comment by rubcap** [comment]
> For anyone interested, I've forked the project and re-published:
> 
> Add-on: https://addons.mozilla.org/en-US/firefox/addon/snooze-tabs-windows-fork/ Source: https://github.com/gmunguia/snoozz-tab-snoozing
> 
> @rohanb10, I'd be happy to collaborate to merge the fork back into this repo.

Thanks, but I don't feel comfortable using your fork without confirmation from @gmunguia why the add-on was removed from AMO in the first place ...
